Dockerfile

```diff
@@ -1,35 +1,5 @@
-# Building node and running it with the customSpecRaw.json
-FROM paritytech/ci-linux:production AS node-builder
-
-WORKDIR /usr/src/node
-
-COPY ./node-template ./
-
-RUN cargo build --release
-
-# Building runtime and generating customSpecRaw.json
-FROM node:14.8 AS builder
-
-WORKDIR /usr/src/
-
-COPY ./runtime ./runtime
-COPY ./node-template/cr-custom-spec.sh ./runtime/tools/spec-builder/cr-custom-spec.sh
-COPY ./node-template/customSpec.json ./runtime/tools/spec-builder/customSpec.json
-
-WORKDIR /usr/src/runtime
-
-RUN yarn install && yarn run build && mv wasm-code ./tools/spec-builder
-
-WORKDIR /usr/src/runtime/tools/spec-builder
-
-RUN apt-get update -y && apt-get install jq -y
-RUN yarn run asbuild && bash cr-custom-spec.sh && yarn build-spec -f customSpec.json
-
-
-FROM ubuntu:18.04
+# Pull node binary from the hub
+FROM limechain/as-substrate-node:stable
 WORKDIR /usr/src/app
-COPY --from=node-builder /usr/src/node/target/release/node-template ./node-template
-COPY --from=builder /usr/src/runtime/tools/spec-builder/customSpecRaw.json ./
-
 EXPOSE 9933 9944/tcp 30333
-ENTRYPOINT ["./node-template", "--chain=./customSpecRaw.json", "--rpc-methods=Unsafe", "--rpc-cors=all", "--rpc-external", "--execution", "Wasm", "--rpc-port", "9933", "--ws-port", "9944", "--port", "30333", "--name", "Node01", "--base-path", "/tmp/node01", "--validator", "--offchain-worker", "Never"]
\ No newline at end of file
+ENTRYPOINT ["./node-template", "--rpc-methods=Unsafe", "--rpc-cors=all", "--rpc-external", "--execution", "Wasm", "--rpc-port", "9933", "--ws-port", "9944", "--port", "30333", "--name", "Node01", "--base-path", "/tmp/node01", "--validator", "--offchain-worker", "Never", "--chain=/customSpecRaw.json"]
```

Comment:
It is purpose defying to have Dockerfile building the node when we are pulling it from an image `limechain/as-substrate-node` and just exposing the ports + defining the arguments. Why are we not using only the `as-substrate-node` image and just provide the custom spec raw. We do not actually need 2 images. 
Let's discuss it